src/call.c

```diff
@@ -2104,6 +2104,19 @@ struct audio *call_audio(const struct call *call)
 }
 
 
+/**
+ * Get the remote IP address for the given call
+ *
+ * @param call  Call object
+ *
+ * @return Remote IP address
+ */
+const struct sa *call_raddr(const struct call *call)
+{
+	return stream_raddr(audio_strm(call_audio(call)));
```

Comment:
what about video stream ?

Reply:
If there is a `c` line in the remote SDP then it is possible that raddr differ for audio and video. But both should be reachable from same local network interface and local address. The result for
```
net_dst_source_addr_get(call_raddr(call),
							     &laddr)
```
should be the same.

The name of the function might be confusing. But we don't need it. I will drop it and use the underlying functions directly.